src/runtime/web/service.js

```diff
@@ -1,39 +1,45 @@
 'use strict';
```

Comment:
why are you moving service to runtime? I thought that runtime is supposed to be code that executes, and Service is still a class with no running code?